### DISCUSSION TITLE
To Linux Users - Don't Waste Time Installing ScummVM to Play Quest Games

### DISCUSSION THREAD

**Discussion by KVonGit** [textadventures/quest#1308] [General]
So, I installed ScummVM in Arch Linux because it runs on Linux and says it will play Quest games. It took about 30 minutes to download and install, but I was thinking it would be worth it the whole time.

![image](https://github.com/user-attachments/assets/6ec3a0c8-27a0-4452-9091-9fef3c91d60b)

![image](https://github.com/user-attachments/assets/c01c6d9c-1387-42f3-9b30-c0962881d829)

![image](https://github.com/user-attachments/assets/671fdf65-fc21-4b76-b218-6a170cab5f06)

---
That's all I read beforehand. Sounds "good." Right?

Well, it wouldn't load any Quest games. So, I searched online again, and found their wiki...

![image](https://github.com/user-attachments/assets/468ead4b-d13f-48a6-9365-7a17d56675cd)

![image](https://github.com/user-attachments/assets/ee28401c-66a9-452b-87e7-bb72015d8e24)

![image](https://github.com/user-attachments/assets/5002c723-856f-4ec1-89e3-8e58e7bbcece)

---
Ha! Hilarious!!!

---
NOTE: There's not really a Discussions category stuff like this fits into. This seemed like the best choice.

**Comment by alexwarren** [comment]
I feel I came across that a while ago and it turns out the "support" is based on "geas" which you can find on the IF Archive. It's a reimplementation from around the time of Quest 3 or so. I recall the guy who wrote it being annoyed because there was no proper spec for the Quest game format.

**Reply by KVonGit** [reply]
Oh, really? If I'd seen that, I'd have known it probably wouldn't work. I found geas in that archive last week and couldn't get it to run because it depends on GLK (and probably a 2001 version of GLK!), and I decided against installing all that.

**Reply by KVonGit** [reply]
Out of the 10 games they list, I found 6. Out of those, the two CAS games are the only ones that will work. It crashes if you try to load an ASL. (This is the opposite of what I would expect.)

Also, trying to play the two games that work in ScummVM is a terrible experience. It prints like 6 lines at a time, and you have to spam the ENTER key to get through the text every turn. Holding ENTER down does not scroll. You have to hit enter line by line to get through a paragraph. Also, no panes (of course).

Two thumbs down! :grin:

**Comment by KVonGit** [comment]
All the games listed as supported:

| Title | Link |
|--|--|
| Adventure! | https://www.ifarchive.org/if-archive/games/quest/adventure.cas |
| Attempted Assassination | https://ifdb.org/viewgame?id=twmyu0ekng2npln1 |
| Beam | https://ifdb.org/viewgame?id=inj3mzj11zjf31mq |
| The Blade Sentinel | https://ifarchive.org/if-archive/games/competition2002/quest/blade/blade%20sentinel.asl |
| Gathered in Darkness | https://ifdb.org/viewgame?id=j737kw3x3j1h8c7e |
| Haunted Horror | https://textadventures.co.uk/games/view/j5p_yjwtnk2skrggjgjmxg/haunted-horror |
| Lovesong | https://textadventures.co.uk/games/view/nwa1gfiyveaf6pqa8bo_ya/love-song |
| Magic World | https://www.ifarchive.org/if-archive/games/quest/Magic.asl |
| Red Sauce Monday | https://www.ifarchive.org/if-archive/games/quest/redsaucemonday.asl |
| World's End | https://www.ifarchive.org/if-archive/games/quest/worldsend.zip |

**Reply by KVonGit** [reply]
I finally found the missing games on if-archive.
